### ISSUE TITLE
segs preprocessor not working

### ISSUE THREAD

**Issue by michel-io** [ltdrdata/ComfyUI-Inspire-Pack#269]
<img width="2914" height="1044" alt="Image" src="https://github.com/user-attachments/assets/02f8aa2c-ac68-4e02-8351-1ed49630f443" />

as you can see, mesgrapmorpher by himself works. but the segs preproprocessor provider never set's the controlnet to the seg

this same problem happends with the DWpreprocessor provider

on the side of the logs it looks like 
`
got prompt
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:16<00:00,  1.19it/s]
Unloaded partially: 2219.93 MB freed, 2677.16 MB remains loaded, 12.50 MB buffer reserved, lowvram patches: 0

0: 640x640 2 hands, 27.0ms
Speed: 4.5ms preprocess, 27.0ms inference, 2.2ms postprocess per image at shape (1, 3, 640, 640)
model_path is F:\ai\ComfyUI-wn\ComfyUI\custom_nodes\comfyui_controlnet_aux\ckpts\hr16\ControlNet-HandRefiner-pruned\graphormer_hand_state_dict.bin
model_path is F:\ai\ComfyUI-wn\ComfyUI\custom_nodes\comfyui_controlnet_aux\ckpts\hr16\ControlNet-HandRefiner-pruned\hrnetv2_w64_imagenet_pretrained.pth
set os.environ[OMP_NUM_THREADS] to 4
=> init weights from normal distribution
=> loading pretrained model F:\ai\ComfyUI-wn\ComfyUI\custom_nodes\comfyui_controlnet_aux\ckpts\hr16\ControlNet-HandRefiner-pruned\hrnetv2_w64_imagenet_pretrained.pth
=> loading pretrained model F:\ai\ComfyUI-wn\ComfyUI\custom_nodes\comfyui_controlnet_aux\ckpts\hr16\ControlNet-HandRefiner-pruned\hrnetv2_w64_imagenet_pretrained.pth
W0000 00:00:1775448856.568450   19300 inference_feedback_manager.cc:114] Feedback manager requires a model with a single signature inference. Disabling support for feedback tensors.
W0000 00:00:1775448856.583474   19300 inference_feedback_manager.cc:114] Feedback manager requires a model with a single signature inference. Disabling support for feedback tensors.
Prompt executed in 45.07 seconds
`

please fix!

**Comment by hiSandog** [comment]
The standalone preprocessor result suggests inference succeeds and the break occurs when its output is passed into SEGS/ControlNet. Could you attach the workflow JSON and exact revisions of Inspire Pack, `comfyui_controlnet_aux`, Impact Pack/SEGS, and ComfyUI? An A/B workflow with only one preprocessor swapped would help determine whether the provider returns the wrong object type, an empty mask, or a value that a newer node version no longer accepts.
